war/src/main/java/io/lumeer/engine/rest/DocumentService.java

```diff
@@ -130,12 +128,16 @@ public void init() {
    @Consumes(MediaType.APPLICATION_JSON)
    public String createDocument(final @PathParam("collectionName") String collectionName, final DataDocument document) throws DbException, InvalidConstraintException, InvalidValueException {
       if (collectionName == null || document == null) {
-         throw new BadRequestException();
+         throw new IllegalArgumentException();
```

Comment:
Why this change? We need to throw `BadRequestException` so RestEasy translates it to HTTP 400.